packages/appeals-service-api/src/lib/questionnaire-due-date.js

```diff
@@ -0,0 +1,36 @@
+const { addDays, endOfDay, format } = require('date-fns');
+
+module.exports = {
+  /**
+   * Given an appeal object, return the due date for the Questionnaire.
+   *
+   * At present, there is only one appeal type - Householder. In future there will be more and this
+   * logic will need to be updated. At the time of creation, `appeal.type` is not a 'thing', and so
+   * we have no easy (i.e. non-brittle) way of determining what type of appeal we are dealing with.
+   *
+   * <b style="color: red;">Therefore all appeals are currently considered as Householder</b>
+   *
+   * @param appeal the appeal object
+   * @returns {Date}
+   */
+  getQuestionnaireDueDate: (appeal) => {
```

Comment:
I can't find anywhere where this is called apart from tests. If this is not called, we don't know whether it will be called at a point in code where the appeal has been checked. Therefore do we need need to test whether the appeal in defined and correct?

Reply:
It's called here: 

https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/1301/files/5f93648451120b4df4051cf30b0838d1bf1dbc3d#diff-453a4d8d442a3c1d16135bad1532a6bf02b8bf69585e39e95b9ded6a768c3adfR35

For clarity:

`getQuestionnaireDueDate` returns a `Date` object. I actually don't want that, it's too 'low level' for my needs in the implementation. However, exporting it makes it nice and easy to test, and also is helpful if anyone ever needs the raw `Date` object.

This is used directly below (line 35) to return a formatted variation that is useful in the implementation.